alyanovikova/Contacts/Sources/Contacts/Contacts.swift

```diff
@@ -0,0 +1,106 @@
+import Foundation
+import Logging
+
+public struct Contact: Codable {
+  public typealias Id = UUID
+
+  public let id: Id
+  public var name: String
+  public var surname: String
+  public var phone: String
+
+  public init(name: String, surname: String, phone: String) {
+    id = Id()
+
+    self.name = name
+    self.surname = surname
+    self.phone = phone
+  }
+}
+
+public class ContactBook {
+  public var contacts = [Contact.Id: Contact]()
+  private let fileURL: URL
+  private let logger = Logger(label: "com.google.WearOS.ContactBook")
+
+  enum ContactError: Error {
+    case nonexistentId
+    case noContact
+  }
+
+  public init() throws {
+    fileURL = try Self.makeDefaultURL()
+    do {
+      contacts = try [Contact.Id: Contact](jsonFileURL: fileURL)
+    } catch CocoaError.fileReadNoSuchFile {
+      contacts = [:]
+      logger.error("FileURL not found, init with blank ContactBook")
+    } catch let error {
```

Comment:
```suggestion
    } catch {
```
Here and in other places